### PR TITLE
Ruby on Rails capstone project: Budget App

### PR DESCRIPTION
**Pull request by shyusu4**
## PR Description

- Created splash screen with links to signup/sign-in using devise gem for authentication
- Created categories page the user can see their name, icon, and the total amount of all the transactions that belongs to that category
- Created a transactions page where the list of transactions is presented, ordered by the most recent
- The user is able to add new categories/transactions
- Created tests for the most important components of the application

Video Presentation: https://www.loom.com/share/3db19c77e7c64e5e98ef2d0d7a41881c

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @shyusu4, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Professional Readme

## Required 
- [ ] Kindly add the video presentation as per the requirements.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @shyusu4, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Image

 ## Required Changes ♻️
- [ ] As suggested by the previous reviewer, kindly add a video presentation as per the requirements.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shyusu4** [comment]
Hi @ShadyShawkat, thank you for the review. I'll make sure to add the video presentation.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,88 @@
...
+> A RoR blog web application that allows users to manage their transactions and sort them by categories.
 
-* System dependencies
+## 🛠 Built With <a name="built-with"></a>
 
-* Configuration
+- Ruby on Rails
 
-* Database creation
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (Ismailco) [inline README.md:25]: - [ ] [OPTIONAL] Kindly make sure that your Live app is working properly as for now it returns an error when You try to log in or signup 👇🏼 

Screenshot_20230120_075830

**Review by Ismailco — APPROVED** [review]
### Hi @shyusu4, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

cr-approve

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**